fxa/oauth.py

```diff
@@ -239,12 +260,39 @@ def verify_token(self, token, scope=None):
             resp = None
 
         if resp is None:
-            url = '/verify'
-            body = {
-                'token': token
-            }
-            resp = self.apiclient.post(url, body)
-
+            # TODO we want to fetch
+            # https://oauth.accounts.firefox.com/.well-known/openid-configuration
+            # and then get the jwks_uri key to get the /jwks url, but we'll
+            # just hardcosde it like this for now until we are closer to
+            # production. We need to do this because oauth.accounts.firefox.com
+            # does not yet respond to .well-known, but accounts.firefox.com
+            # does; for consistency, oauth.accounts.firefox.com should respond
+            # to .well-known.
```

Comment:
We should do this now.

Reply:
@dannycoates says the jwks url isn't going to change before this python codebase is retired... So maybe we don't need to do this.

Reply:
That's true for tokenserver but maybe not true for all consumers of this library. Still, I think I'd be happy not to block on it in the short-term.

Reply:
I'd be happy to merge this with a re-worded comment that removes the "closer to production" stuff in favor of something a little more definitive.

Alternately, we could consider a speculative fetch here that looks something like:

```
try:
  settings = self.apiclient.get("/.well-known/openid-configuration")
except CatchOnly404NotFoundIForgetTheRightSyntax:
  settings = {"jwks_uri": "/jwks"}
```

I don't have strong feelings either way.

Reply:
I'd like to leave it hardcoded and open a new bug for fetching the openid-configuration. Making a build that only does one http request instead of two for us to use until the tokenserver is ported to rust will help alleviate load a tiny bit. Since alleviating load is the entire point of this work, I think it's worth it for now.

Reply:
https://github.com/mozilla/PyFxA/issues/81